tensorboard/components_polymer3/vz_sorting/BUILD

```diff
@@ -1,17 +1,16 @@
 load("//tensorboard/defs:defs.bzl", "tensorboard_webcomponent_library")
-load("//tensorboard/defs:web.bzl", "tf_web_library")
+load("//tensorboard/defs:defs.bzl", "tf_ts_library")
 
 package(default_visibility = ["//tensorboard:internal"])
 
 licenses(["notice"])  # Apache 2.0
 
-tf_web_library(
+tf_ts_library(
     name = "vz_sorting",
     srcs = [
         "sorting.ts",
-        "vz-sorting.html",
     ],
-    path = "/vz-sorting",
+    strict_checks = False,
 )
 
 tensorboard_webcomponent_library(
```

Comment:
remove me. I don't think internal rules will be able to handle this dependency.

Reply:
Done